package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/lua
+
+function restartTunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
```

Comment:
wouldn't a print statement suffice here to make it show up in logread?

Reply:
I'm not sure, would it?

Reply:
Nah, micron does not seem to log stdout/stderr to syslog.

Reply:
Jup, just tried it myself, doesn't work :)